README.md

```diff
@@ -25,10 +25,19 @@ To address challenges 5 and 6, this repository contains a freely accessible, une
 
 ![A wallet architecture using Hierarchical Deterministic Keys associated with keys protected using a wallet secure cryptographic device, optionally using Asynchronous Remote Key Generation (ARKG).](media/deployment.svg)
 
+Please note that when ARKG is used, issuers have to store an additional secret key per user / holder.
+
 To illustrate and validate the specifications, this repository contains a **[Prototype worksheet](prototype.worksheet.sc)**. This is easiest to run in [Visual Studio Code](https://code.visualstudio.com) with [Scala (Metals)](https://marketplace.visualstudio.com/items?itemName=scalameta.metals).
 
 To inform further standardisation and legislation, this repository contains **[Feedback to enable Hierarchical Deterministic Keys in the Wallet Toolbox](feedback.md)**.
 
+This document does not contain details about the implementation of HDK type pseudonymous random generation for BBS# key management because the author prioritized HDK. However, [BBS#](https://github.com/user-attachments/files/15905230/BBS_Sharp_Short_TR.pdf) would in theory present many advantages over the proposed HDK implementations:
+- a single parameter is needed (a random), not 3 (public_key, private_key, salt)
+- only local derivation is needed (no need for remote derivation)
+- seed sharing between holder and issuer is not needed ==> much reduced overhead of the issuer (no need to store and secure one seed per holder)
+- BBS# is designed from the ground up for full unlinkability which is not achievable with only HDK
+- Privacy and security proofs are provided
+
```

Comment:
```suggestion
```

OK to leave this discussion for a next iteration? As I see it, local HDKs could provide benefit to BBS# solutions, but this needs further discussion. Taking it in scope would delay publication.

Reply:
No, either we put the whole thing in perspective or we will not sign it. You can happily keep the rest of my contributions for the document as you like but if this is whole very complex HDK + ARKG is not put in perspective with the fact that BBS# could theoretically offer more in a much simpler way, please remove my name from the supporters.

Reply:
Counter-proposal posted in Signal, to indeed appreciate the broader perspective that HDK is not enough and a credential scheme is needed, which is a logical continuation for the working group.

> The repository does not contain details about the implementation of HDK for key management in credential schemes, such as one-time-use document schemes for relying party unlinkability and weak issuer unlinkability, or [BBS#](https://github.com/user-attachments/files/15905230/BBS_Sharp_Short_TR.pdf) for full unlinkability. Credential schemes require further elaboration by the participants. This may lead to changes to the specification. For example, delegated key generation only seems to have use cases for batch one-time-use document issuance.